### DISCUSSION TITLE
Rectangle settings window not resizable

### DISCUSSION THREAD

**Discussion by Kinrany** [rxhanson/Rectangle#709] [Ideas]
- [x] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: Big Sur 11.5.1
Rectangle version: 0.48

When you start Rectangle for the first time, a settings window pops up. It lists and allows you to change all the hotkeys.

It seems natural to try all the hotkeys by moving the settings window around. However the window is not resizable, which is very confusing: half the hotkeys do not work or do not do what you expect.

**Comment by rxhanson** [comment]
I agree that this is confusing. Unfortunately, allowing the size to change could end up looking pretty weird. I'm ok with the window the way it is, so I don't have a desire to change it. But, if someone comes up with a good solution to this I'd be up for merging a good PR.
